rust/gitxet/build.rs

```diff
@@ -2,6 +2,8 @@
 use std::env;
 
 fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
do we want this println?

Reply:
nevermind, I just found out you need to print this line to compile on change only, a bit surprised it's a println not another macro.